Casks/e/electorrent.rb

```diff
@@ -7,6 +7,13 @@
   desc "Desktop remote torrenting application"
   homepage "https://github.com/tympanix/Electorrent"
 
+  livecheck do
+    url "https://electorrent.vercel.app/update/dmg/0.0.0"
+    strategy :json do |json|
+      json["name"]&.tr("v", "")
```

Comment:
```suggestion
    regex(/^v?(\d+(?:\.\d+)+)$/i)
    strategy :json do |json|
      json["name"]&.[](regex, 1)
```

Removing `v` from the `name` string works for now but it wouldn't work as expected if the string uses a capital `V` or if `v` appears elsewhere in the string. It's probably better to be a little more explicit that we're omitting a `v` before the version and to do it in a case insensitive way. Assuming upstream sticks to the `v1.2.3` format for stable versions, we can simply use the standard regex for versions like this.